webknossos-datastore/proto/Annotation.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto2";
+
+package com.scalableminds.webknossos.datastore;
+
+enum AnnotationLayerTypeProto {
+  Skeleton = 1;
+  Volume = 2;
+}
+
+message AnnotationProto {
+  optional string name = 1;
+  optional string description = 2;
+  required int64 version = 3;
+  repeated AnnotationLayerProto layers = 4;
```

Comment:
@fm3 during frontend implementation I noticed that this field is called `annotationLayers` in the annotation info object. Is it maybe possible to unify this naming :thinking: 

Although it might not be that important as the info on layers from the annotation proto always overwrites the info within the annotation info object

Reply:
sure, we can rename it!

Reply:
i renamed it to annotationLayers in the proto definition and changed the two frontend usages I found